src/GraphQL.Tests/Utilities/SchemaBuilderExecutionTests.cs

```diff
@@ -300,6 +301,7 @@ public void can_use_context_source_usercontext()
         public void can_use_context_source_usercontext_with_params()
         {
             var schema = Schema.For(@"
+                input
```

Comment:
I believe this is probably causing tests to fail.

Reply:
Missed it, thanks.